### PR TITLE
 Pbf2Atlas command

### PR DESCRIPTION
**Pull request by matthieun**
### Description:

Added a command that creates a way-sectioned Atlas from a pbf file.
- Also revamped some of the JTS converters that were lacking to enable the command to filter based on a WKT multipolygon.

### Potential Impact:

New command available

### Unit Test Approach:

Added unit tests to the JTS converters

### Test Results:

Tests pass

------

In doubt: [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

**Comment by matthieun** [comment]
Rhaa sonar is right. I will fix those and reopen.

File: src/main/java/org/openstreetmap/atlas/geography/converters/WkMultiPolygonConverter.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import org.openstreetmap.atlas.geography.converters.jts.JtsMultiPolygonToMultiPolygonConverter;
+import org.openstreetmap.atlas.geography.converters.jts.JtsPolygonToMultiPolygonConverter;
+import org.openstreetmap.atlas.utilities.conversion.TwoWayConverter;
+
+/**
+ * @author matthieun
+ * @param <T>
+ *            The type to convert to: {@link String} or byte[]
+ */
+public abstract class WkMultiPolygonConverter<T> implements TwoWayConverter<MultiPolygon, T>
```
Comment (jwpgage) [inline src/main/java/org/openstreetmap/atlas/geography/converters/WkMultiPolygonConverter.java]: nit: WkMultiPolygonConverter -> WktMultiPolygonConverter

Comment (jwpgage) [reply src/main/java/org/openstreetmap/atlas/geography/converters/WkMultiPolygonConverter.java]: oh, I see we have a class with that name already...

Comment (jwpgage) [inline src/main/java/org/openstreetmap/atlas/geography/converters/WkMultiPolygonConverter.java]: Nice refactor! The names had me confused for a sec :)

Comment (lucaspcram) [reply src/main/java/org/openstreetmap/atlas/geography/converters/WkMultiPolygonConverter.java]: Wait what is the significance of the `Wk`?

Edit: Oh never mind I understand now. This is a superclass for `WKT` or `WKB`. Maybe add that to the javadoc? :)

File: src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/PbfToAtlasCommand.java [hunk 2]
```diff
@@ -21,16 +26,25 @@
 
 /**
  * @author samgass
+ * @author matthieun
  */
-public class PbfToRawAtlasCommand extends MultipleOutputCommand
+public final class PbfToAtlasCommand extends MultipleOutputCommand
```
Comment (lucaspcram) [inline src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/PbfToAtlasCommand.java]: Woohoo!

**Pull request merged**